openpype/modules/ftrack/plugins/publish/integrate_ftrack_instances.py

```diff
@@ -147,6 +151,8 @@ def process(self, instance):
         first_thumbnail_component = None
         first_thumbnail_component_repre = None
         for repre in thumbnail_representations:
+            if review_representations and not has_movie_review:
+                break
```

Comment:
Both these variables don't seem to be related to the `repre` we are currently iterating and thus it feels off that we're checking these variables inside the for loop instead of prior to it? (It looks confusing because it reads as if related to the current repre being iterated)